package.json

```diff
@@ -91,9 +94,13 @@
     "@nx/workspace": "16.3.1",
     "@storybook/html-webpack5": "^7.0.23",
     "@storybook/react-webpack5": "^7.0.23",
+    "@testing-library/react": "^14.0.0",
```

Comment:
Should we also consider moving `@testing-library/jest-dom` to root?

Reply:
There are a bunch of subdependencies of @testing-library/* that should be moved, along with the centralization of our jest versions. I'd like to consider that out of scope for this, however.